doc/references.bib

```diff
@@ -0,0 +1,453 @@
+  author={Havlicek, Vojtech and C{\'o}rcoles, Antonio D and Temme, Kristan and Harrow, Aram W and Chow, Jerry M and Gambetta, Jay M},
+  year={2018},
+  note = {},
+  archivePrefix = {arxiv},
+  eprint = {1804.11326}
+}
+
+@unpublished{fingerhuth2018quantum,
+  title={A quantum alternating operator ansatz with hard and soft constraints for lattice protein folding},
+  author={Fingerhuth, Mark and Babej, Tom{\'a}{\v{s}} and others},
+  year={2018},
+  note = {},
+  archivePrefix = {arxiv},
+  eprint = {1810.13411}
+}
+
+@article{shepherd2009temporally,
+  doi = {10.1098/rspa.2008.0443},
+  year  = {2009},
+  publisher = {The Royal Society},
+  volume = {465},
+  number = {2105},
+  pages = {1413--1439},
+  author = {D. Shepherd and M. J. Bremner},
+  title = {Temporally unstructured quantum computation},
+  journal = {Proceedings of the Royal Society A: Mathematical, Physical and Engineering Sciences}
+}
+
+@unpublished{arrazola2017quantum,
+  title={Quantum supremacy and high-dimensional integration},
+  author={Arrazola, Juan Miguel and Rebentrost, Patrick and Weedbrook, Christian},
+  year={2017},
+  note = {},
+  archivePrefix = {arxiv},
+  eprint = {1712.07288}
+}
+
+@inproceedings{miles2016supervised,
+  title = {Supervised Learning with Tensor Networks},
+  author = {Stoudenmire, Edwin and Schwab, David J},
+  booktitle = {Advances in Neural Information Processing Systems 29},
+  editor = {D. D. Lee and M. Sugiyama and U. V. Luxburg and I. Guyon and R. Garnett},
+  pages = {4799--4807},
+  year = {2016},
+  publisher = {Curran Associates, Inc.},
+  url = {http://papers.nips.cc/paper/6211-supervised-learning-with-tensor-networks.pdf}
+}
+
+@article{lloyd2018quantum,
+  doi = {10.1103/physrevlett.121.040502},
+  year  = {2018},
+  publisher = {American Physical Society ({APS})},
+  volume = {121},
+  number = {4},
+  author = {Seth Lloyd and Christian Weedbrook},
+  title = {Quantum Generative Adversarial Learning},
+  journal = {Physical Review Letters}
+}
+
+@article{clements2016optimal,
+  doi = {10.1364/OPTICA.3.001460},
+  year  = {2016},
+  publisher = {OSA Publishing},
+  volume = {3},
+  number = {12},
+  pages = {1460--1465},
+  author = {William R. Clements and Peter C. Humphreys and Benjamin J. Metcalf and W. Steven Kolthammer and Ian A. Walmsley},
+  title = {An Optimal Design for Universal Multiport Interferometers},
+  journal = {Optica}
+}
+
+@article{reck1994experimental,
+  doi = {10.1103/PhysRevLett.73.58},
+  title={Experimental realization of any discrete unitary operator},
+  author={Reck, Michael and Zeilinger, Anton and Bernstein, Herbert J and Bertani, Philip},
+  journal={Physical review letters},
+  volume={73},
+  number={1},
+  pages={58},
+  year={1994},
+  publisher={APS}
+}
+
+@book{schuld2018supervised
+  title = {Supervised Learning with Quantum Computers},
+  SUBTITLE = {Quantum Science and Technology },
+  AUTHOR = {Schuld, Maria and Petruccione, Francesco},
+  YEAR = {2018}, 
+  PUBLISHER = {Springer},
+}
```

Comment:
In the QML/PennyLane repository split, PennyLane lost the bib file :) The bib file is now part of the QML repo, since it hosts the background information/theory.

As a result, docstrings in the PennyLane docs now simply have a handwritten reference/link to the arXiv.

So best to remove this file, and remove any references that were added.